src/System.Collections.Specialized/src/System/Collections/Specialized/NameObjectCollectionBase.cs

```diff
@@ -90,56 +90,12 @@ protected NameObjectCollectionBase(int capacity)
 
         protected NameObjectCollectionBase(SerializationInfo info, StreamingContext context)
         {
-            _serializationInfo = info;
+            throw new PlatformNotSupportedException();
```

Comment:
I assume your tool to analyze common serialized types factored in base types?  I'm wondering about how common it is to serialize a type derived from NameObjectCollectionBase.